WordPress/Classes/Jetpack/JetpackAuthenticationManager.swift

```diff
@@ -3,7 +3,7 @@ import UIKit
 
 struct JetpackAuthenticationManager: AuthenticationHandler {
     let statusBarStyle: UIStatusBarStyle = .default
-    let prologueViewController: UIViewController? = JetpackPrologueViewController()
+    let prologueViewController: UIViewController? = JetpackPrologueViewController(nibName: "JetpackPrologueViewController", bundle: .keystone)
```

Comment:
Nitpick: What do you think of generating the string from the class?

Like with `String(describing: type(of: self)`?

I like this approach because if the unlikely case (hence the nitpick) that we'll rename the class, the string used here should update automatically and accordingly.